coherent_rates/isf.py

```diff
@@ -298,6 +301,41 @@ def _get_boltzmann_isf_from_hamiltonian(
     }
 
 
+def get_coherent_thermal_state(
+    system: PeriodicSystem,
+    config: PeriodicSystemConfig,
+    sigma_0: float,
+) -> StateVector[Any]:
+    # generates a Gaussian state with x0,k0 given approximately by a thermal distribution
```

Comment:
this should be a doc comment, not an inline comment because it's for the people using the function (describing what rather than how)